src/components/PostList.tsx

```diff
@@ -1,9 +1,37 @@
 import React from "react";
+import { useAppSelector } from "../redux/hooks";
+import { FeedItem } from "../utilities/types";
+import { NoteActivityPub } from "../utilities/activityPubTypes";
 
 const PostList: React.FC = () => {
+  const feed: FeedItem[] = useAppSelector((state) => state.feed.feed);
+  feed.filter((post) => post?.content?.type === "Note");
```

Comment:
It looks like this is not being set to a variable so its not being used.

I think you meant to do something like
```
  const feed: FeedItem[] = 
     useAppSelector((state) => state.feed.feed).filter((post) => post?.content?.type === "Note")
 ```

Reply:
From checking out the snapshot, it seems like the opposite.  Maybe we can pair on this.